packages/web-components/src/global/colors.css

```diff
@@ -0,0 +1,521 @@
+  );
+  --ic-status-warning-hover: rgb(
+    calc(var(--ic-status-warning-r) * 0.8)
+      calc(var(--ic-status-warning-g) * 0.8)
+      calc(var(--ic-status-warning-b) * 0.8) / var(--ic-status-warning-a)
+  );
+  --ic-status-warning-pressed: rgb(
+    calc(var(--ic-status-warning-r) * 0.6)
+      calc(var(--ic-status-warning-g) * 0.6)
+      calc(var(--ic-status-warning-b) * 0.6) / var(--ic-status-warning-a)
+  );
+
+  /* error status */
+  --ic-status-error-default: var(--ic-red-800);
+  --ic-status-error-light: var(--ic-red-0);
+  --ic-status-error-dark: var(--ic-red-1500);
+  --ic-status-error-outlined: var(--ic-red-500);
+  --ic-status-error-contrast: var(--ic-red-400);
+  --ic-status-error-r: 212;
+  --ic-status-error-g: 53;
+  --ic-status-error-b: 28;
+  --ic-status-error-a: 1;
+  --ic-status-error: rgb(
+    var(--ic-status-error-r) var(--ic-status-error-g) var(--ic-status-error-b) /
+      var(--ic-status-error-a)
+  );
+  --ic-status-error-hover: rgb(
+    calc(var(--ic-status-error-r) * 0.8) calc(var(--ic-status-error-g) * 0.8)
+      calc(var(--ic-status-error-b) * 0.8) / var(--ic-status-error-a)
+  );
+  --ic-status-error-pressed: rgb(
+    calc(var(--ic-status-error-r) * 0.6) calc(var(--ic-status-error-g) * 0.6)
+      calc(var(--ic-status-error-b) * 0.6) / var(--ic-status-error-a)
+  );
+
+  /* info status */
+  --ic-status-info-default: var(--ic-blue-900);
+  --ic-status-info-light: var(--ic-blue-0);
+  --ic-status-info-dark: var(--ic-blue-1600);
+  --ic-status-info-contrast: var(--ic-blue-400);
+
+  /* anomalous status */
+  --ic-status-anomalous-default: var(--ic-purple-1400);
+  --ic-status-anomalous-light: var(--ic-purple-50);
+  --ic-status-anomalous-dark: #350f54;
+  --ic-status-anomalous-contrast: var(--ic-purple-500);
+
+  /* status unknown */
+  --ic-status-unknown-default: var(--ic-architectural-500);
+  --ic-status-unknown-mid: var(--ic-architectural-700);
+  --ic-status-unknown-light: var(--ic-architectural-40);
+  --ic-status-unknown-dark: var(--ic-architectural-800);
+  --ic-status-unknown-outlined: var(--ic-architectural-200);
+  --ic-status-unknown-contrast: var(--ic-architectural-300);
+
+  /* TEXT */
+  --ic-color-text-primary-light: var(--ic-architectural-900);
+  --ic-color-text-primary-dark: var(--ic-architectural-white);
+  --ic-color-text-secondary-light: var(--ic-architectural-700);
+  --ic-color-text-secondary-dark: var(--ic-architectural-100);
+  --ic-color-text-tertiary-light: var(--ic-architectural-500);
+  --ic-color-text-tertiary-dark: var(--ic-architectural-200);
+  --ic-color-text-inverse-light: var(--ic-color-text-primary-dark);
+  --ic-color-text-inverse-dark: var(--ic-color-text-primary-light);
+  --ic-color-brand-text: var(--ic-architectural-white);
+  --ic-color-white-text: var(--ic-architectural-white); /* to be removed */
+  --ic-color-text-disabled-light: var(--ic-architectural-300);
+  --ic-color-text-disabled-dark: var(--ic-architectural-600);
```

Comment:
Should be --ic-architectural-200?

Reply:
`--ic-color-text-disabled-dark` should be `var(--ic-architectural-600)` according to the Figma tokens
 <img width="390" alt="Screenshot 2024-10-08 at 10 04 37" src="https://github.com/user-attachments/assets/843ded5f-88d9-44bd-bd51-0465dc998a97">